src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -81,7 +82,7 @@ var ReactDOMInput = {
       onChange: inst._wrapperState.onChange,
     });
 
-    return nativeProps;
+    return DisabledInputUtils.getNativeProps(inst, nativeProps);
```

Comment:
I'm playing around with an idea to try to save that object allocation, but to directly reply to your comment. It should be consistent with the other methods. I'll do that now.